File: app/src/main/java/ru/visualmath/android/api/model/Results.java
package ru.visualmath.android.api.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

public class Results implements Serializable {

    @SerializedName("answers")
    private List<Answer> answers;

    @SerializedName("sum")
    private double sum;

    public List<Answer> getAnswers() {
        return answers;
    }

    public double getSum() {
        return sum;
    }

    @Override
    public String toString() {
        return "Results{" +
                "answers=" + answers +
                ", sum=" + sum +
                '}';
    }
}

File: app/src/main/java/ru/visualmath/android/api/model/Results.kt
package ru.visualmath.android.api.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Results(

        @SerializedName("answers")
        val answers: List<Answer> = emptyList(),

        @SerializedName("sum")
        val sum: Double = 0.0
) : Parcelable
